SeaPublicWebsite.BusinessLogic/Models/PropertyData.cs

```diff
@@ -44,6 +44,56 @@ public class PropertyData
     public int? HoursOfHeatingMorning { get; set; }
     public int? HoursOfHeatingEvening { get; set; }
     public decimal? Temperature { get; set; }
+    public PropertyData Backup { get; set; }
 
     public List<PropertyRecommendation> PropertyRecommendations { get; set; }
+
+    public void CreateBackup()
+    {
+        Backup = new PropertyData();
+        CopyAnswersTo(Backup);
+    }
+    
+    public void ApplyBackup()
+    {
+        Backup.CopyAnswersTo(this);
+    }
+    
+    public void ResetBackup()
+    {
+        Backup = null;
+    }
+
+    private void CopyAnswersTo(PropertyData other)
+    {
+        other.OwnershipStatus = OwnershipStatus;
+        other.Country = Country;
+        other.Epc = Epc;
```

Comment:
Let me know if you have a nicer way to do this (I tried iterating through the object's properties but it feels hacky)